### PR TITLE
feature/story_46943_plugin_portal_task

### PR DESCRIPTION
**Pull request by Hari-Ritense**

### CODE REVIEW

File: getting-started/modules/zgw/portaaltaak.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Notificaties API
```
Comment (Klaas-Ritense) [inline getting-started/modules/zgw/portaaltaak.md:1]: Wrong header

File: getting-started/modules/zgw/portaaltaak.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
```
Comment (Klaas-Ritense) [inline getting-started/modules/zgw/portaaltaak.md:2]: Maybe add a description like:
This plugin makes it possible for Valtimo to communicate and interact with a Portal task specification compliant ZGW component.

File: getting-started/modules/zgw/portaaltaak.md (hunk 1 above)
Comment (Hari-Ritense) [reply getting-started/modules/zgw/portaaltaak.md:1]: Smart Klaas. Ga aanpassen.

File: getting-started/modules/zgw/portaaltaak.md (hunk 2 above)
Comment (Hari-Ritense) [reply getting-started/modules/zgw/portaaltaak.md:2]: Are you sure? I don't see the description part in other modules.md as well.

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**